build.xml

```diff
@@ -1148,9 +1148,26 @@ ext = BeanUtils.cook(ext);
         <unzip dest='${webapp.dir}' src="${webapp.file}"/>
 
         <antcall target='webxml-deploy-replace'/>
+        <antcall target='timezones-deploy'/>
```

Comment:
build script is using prod-war target
and we are modifying deploy target, i guess deploy is only used for deploying war in dev setup